docs/cse/ingestion/ingestion-sources-for-cloud-siem/palo-alto-firewall.md

```diff
@@ -101,7 +101,7 @@ To create a server profile specifying the log destination, do the following:
 
 [GlobalProtect](https://docs.paloaltonetworks.com/pan-os/10-0/pan-os-admin/monitoring/use-syslog-for-monitoring/syslog-field-descriptions/globalprotect-log-fields)
 ```
-,$receive_time,$serial,$type,$subtype,,$time_generated,$vsys,$ip,$user,$datasourcename,$eventid,$repeatcnt,$timeout,$beginport,$endport,$datasource,$datasourcetype,$seqno,$actionflags,$dg_hier_level_1,$dg_hier_level_2,$dg_hier_level_3,$dg_hier_level_4,$vsys_name,$device_name,$vsys_id,$factortype,$factorcompletiontime,$factorno,,,$ugflags,$userbysource
+,$serial,$type,$subtype,$time_generated,$vsys,$eventid,$stage,$auth_method,$tunnel_type,$src_user,$srcregion,$machinename,$public_ip,$public_ipv6,$private_ip,$private_ipv6,$hostid,$serialnumber,$client_ver,$client_os,$client_os_ver,$repeatcnt,$reason,$error,$opaque,$status,$location,$login_duration,$connect_method,$error_code,$portal,$seqno,$actionflags
```

Comment:
@mvirga-sumo we'll await your final signoff before merging

Reply:
@kimsauce  I have submitted the final PR that is comprehensive for this doc - that is PR #2875